frontend/src/components/views/Login/Login.jsx

```diff
@@ -61,6 +61,34 @@ class Login extends Component {
         alert('error has occured');
       }
     };
+
+    this.handleLoginRd = async (event) => {
```

Comment:
can use rider***

Reply:
ok! do i do this for all the other rider methods as well? and because for the methods with managers i used mg as short for manager, do you need me to make changes to the method names for managers also?

Reply:
sure, we abbreviate only when it gets to long bah!

Reply:
okiee